src/models/multiBarChart.js

```diff
@@ -16,7 +16,7 @@ nv.models.multiBarChart = function() {
     var margin = {top: 30, right: 20, bottom: 50, left: 60}
         , width = null
         , height = null
-        , color = nv.utils.defaultColor()
+        , color = nv.utils.getColor(function(d,i) {return d3.rgb('#ccc').darker(i * 1.5).toString();})
```

Comment:
This should be done in the barColor() setter, the overall default should remain nv.utils.defaultColor() since barColor() is not the default.